docs/OCP_CI_Tutorials/Scenarios/Scenarios_Guide.md

```diff
@@ -17,18 +17,45 @@
 ## Overview
 The following Headers are phases that MUST be done for each scenario. They are needed to provide a structure to the many OpenShift layered product scenarios that will be running on OpenShift CI.
 
+Below is a basic flow chart of how a OpenShift CI scenario may look. Each scenario should have a chain where most of the work is completed. This chain should be named something like `interop-{layered-product name}-chain`. The scenario chain should execute the orchestrate, execute, and report steps in order. The orchestrate, execute, and report steps of a scenario's chain can either be another chain or a single ref depending on your scenario's specific needs.
+
+```mermaid
+---
+title: Basic Scenario Structure
+---
+flowchart LR
+  scenario((interop-scenario-chain)) --> orchestrate_step --> execute_step --> report_step
+
+  subgraph orchestrate_step
+    orchestrate_ref1([interop-orchestrate-ref]) --> orchestrate_ref2([interop-tooling-ref]) --> orchestrate_ref3([interop-some-other-ref])
+  end
+  
+  subgraph execute_step
+    execute_ref1([interop-execute-ref])
+  end
+
+  subgraph report_step
+    report_ref1([interop-tooling-report-ref])
+  end
+
+```
 ## Provision
-We define provision as the initial setup of your scenarios files and the OpenShift installation
+We define provision as:
+> The initial setup of your scenarios files and the OpenShift installation
 
 See the [Provision Guide](Provision_Guide.md) for detailed information meant to help you build the foundation of your scenario.
 
 ## Orchestrate
-We define Orchestrate as anything, post OpenShift install, that is needed to install and configure your product on OpenShift so that it is ready to be tested.
+We define Orchestrate as:
```

Comment:
In order to maintain consistency with the provision section, "Orchestrate" should be "orchestrate"

Reply:
Resolved in this commit: https://github.com/CSPI-QE/ocp-ci-docs/pull/7/commits/56f31cb3e8717dd8ff3195cc49369f2fd49b16bf